<!DOCTYPE html>
<html><head><title>ErrorSourceType</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">11 references to ErrorSourceType</div><div class="rA">Microsoft.Azure.EventHubs (9)</div><div class="rG" id="Microsoft.Azure.EventHubs"><div class="rF"><div class="rN">Primitives\EventHubsCommunicationException.cs (1)</div>
<a href="../Primitives/EventHubsCommunicationException.cs.html#24"><b>24</b>: base(true, message, innerException, <i>ErrorSourceType</i>.UserError)</a>
</div>
<div class="rF"><div class="rN">Primitives\EventHubsException.cs (2)</div>
<a href="../Primitives/EventHubsException.cs.html#79"><b>79</b>public EventHubsException(bool isTransient, string message, Exception innerException, <i>ErrorSourceType</i> errorSource)</a>
<a href="../Primitives/EventHubsException.cs.html#117"><b>117</b>public <i>ErrorSourceType</i> ErrorSource { get; private set; }</a>
</div>
<div class="rF"><div class="rN">Primitives\EventHubsTimeoutException.cs (1)</div>
<a href="../Primitives/EventHubsTimeoutException.cs.html#23"><b>23</b>: base(true, message, innerException, <i>ErrorSourceType</i>.ServerError)</a>
</div>
<div class="rF"><div class="rN">Primitives\MessageSizeExceededException.cs (1)</div>
<a href="../Primitives/MessageSizeExceededException.cs.html#19"><b>19</b>: base(false, message, innerException, <i>ErrorSourceType</i>.UserError)</a>
</div>
<div class="rF"><div class="rN">Primitives\MessagingEntityNotFoundException.cs (1)</div>
<a href="../Primitives/MessagingEntityNotFoundException.cs.html#12"><b>12</b>: base(false, message, null, <i>ErrorSourceType</i>.UserError)</a>
</div>
<div class="rF"><div class="rN">Primitives\QuotaExceededException.cs (1)</div>
<a href="../Primitives/QuotaExceededException.cs.html#24"><b>24</b>: base(false, message, innerException, <i>ErrorSourceType</i>.ServerError)</a>
</div>
<div class="rF"><div class="rN">Primitives\ReceiverDisconnectedException.cs (1)</div>
<a href="../Primitives/ReceiverDisconnectedException.cs.html#20"><b>20</b>: base(false, message, innerException, <i>ErrorSourceType</i>.UserError)</a>
</div>
<div class="rF"><div class="rN">Primitives\ServerBusyException.cs (1)</div>
<a href="../Primitives/ServerBusyException.cs.html#19"><b>19</b>: base(true, message, innerException, <i>ErrorSourceType</i>.ServerError)</a>
</div>
</div>
<div class="rA">Microsoft.Azure.EventHubs.Processor (2)</div><div class="rG" id="Microsoft.Azure.EventHubs.Processor"><div class="rF"><div class="rN">EventProcessorConfigurationException.cs (1)</div>
<a href="../../Microsoft.Azure.EventHubs.Processor/EventProcessorConfigurationException.cs.html#19"><b>19</b>: base(false, message, innerException, <i>ErrorSourceType</i>.UserError)</a>
</div>
<div class="rF"><div class="rN">EventProcessorRuntimeException.cs (1)</div>
<a href="../../Microsoft.Azure.EventHubs.Processor/EventProcessorRuntimeException.cs.html#19"><b>19</b>: base(true, message, innerException, <i>ErrorSourceType</i>.UserError)</a>
</div>
</div>
</body></html>